pymc/logprob/abstract.py

```diff
@@ -153,3 +155,29 @@ def __init__(self, scalar_op, *args, **kwargs):
 
 
 MeasurableVariable.register(MeasurableElemwise)
+
+
+@multipledispatch(Op, Op)
+def _kl_div(
+    q_dist: Op,
+    p_dist: Op,
+    q_inputs: List[TensorVariable],
+    p_inputs: List[TensorVariable],
```

Comment:
This is wrong, RVs have non tensor inputs as well
```suggestion
    q_inputs: List[Variable],
    p_inputs: List[Variable],
```